docs/source/science/GEDI/GEDI_L4B.ipynb

```diff
@@ -2,33 +2,80 @@
  "cells": [
```

Comment:
<p>If we're using the concept-id, I don't think that limit is necessary?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/262/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>